.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: "CI"
+
+on: [push,pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
```

Comment:
Please update to support both Node 10 and 12 like this: https://github.com/paypal/react-paypal-js/blob/main/.github/workflows/main.yml#L11.